changelog.txt

```diff
@@ -2,6 +2,10 @@ STELVIO CHANGELOG
 =================
 https://confluence.adeo.no/display/stelvio/Application+Framework+-+Change+Log
 
+Version 6.1.0

```

Comment:
Burde dette vært 7.0?

Reply:
@pcmoen hva synes du? Hvis man ser i historikken her, så har gjerne Spring og Hibernate vært gjort samtidig når man har gått til ny major. For min del kan vi fint gjøre det - da blir Spring 5 8.0.

Reply:
@pcmoen hva synes du? Hvis man ser i historikken her, så har gjerne Spring og Hibernate vært gjort samtidig når man har gått til ny major. For min del kan vi fint gjøre det - da blir Spring 5 8.0.

Reply:
Ja kan være lurt. Så ser man lettere at det er en større endring.